packages/run-protocol/test/runStake/runStake-test-sheet.js

```diff
@@ -0,0 +1,200 @@
+// @ts-check
+/**
+ * Test steps with support for spreadsheet integration.
+ */
+import util from 'util';
+
+// ack: https://stackoverflow.com/a/18147076/7963
+const CSV = (() => {
+  const fieldPattern = /("(?:(?:"")*[^"]*)*"|[^",\n]*)(?:,|\n|$)/g;
+
+  /** @param { string } field */
+  const unquote = field =>
+    field.replace(/""/g, '"').replace(/^"/, '').replace(/"$/, '');
+
+  /** @param { string } line */
+  const splitFields = line =>
+    [...line.matchAll(fieldPattern)].map(([_all, txt]) => unquote(txt));
+
+  /** @param { string } content */
+  const parse = content => content.split('\n').map(splitFields);
+
+  return harden({ parse });
+})();
+
+// copy/paste from spreadsheet seems to give tab-separated data
```

Comment:
can the spreadsheet be linked here?

Reply:
In case you're curious, it's linked from #3788 . But I no longer maintain the tests this way. (more below...)